### PR TITLE
feat: merge develop

### PR DESCRIPTION
**Pull request by BC-A**

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "## How to use",
+  "main": "config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "link:lib": "npm link @0xEigenLabs/eigen-sdk-js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@maticnetwork/maticjs-web3": "^1.0.4",
```
Comment (eigmax) [inline examples/package.json:13]: Can we change this to web3.js?  what's  the difference between them?

File: src/enums/log_event_signature.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export enum Log_Event_Signature {
```
Comment (eigmax) [inline src/enums/log_event_signature.ts:1]: What are the constants used for?

Comment (eigmax) [reply src/enums/log_event_signature.ts:1]: Transfer function's signature?

File: src/implementation/bn.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { throwNotImplemented } from "..";
```
Comment (eigmax) [inline src/implementation/bn.ts:1]: Is this file used for?

File: src/enums/log_event_signature.ts (hunk 2 above)
Comment (JangoCCC) [reply src/enums/log_event_signature.ts:1]: yes, hard-coded transfer function signature.

File: src/implementation/bn.ts (hunk 3 above)
Comment (JangoCCC) [reply src/implementation/bn.ts:1]: This file just implements an EmptyBigNumber class. The real implementation is in the plugin instance.

**Pull request merged**